botocore/utils.py

```diff
@@ -1085,3 +1085,54 @@ def _get_response(self, full_url, headers, timeout):
 
     def full_url(self, relative_uri):
         return 'http://%s%s' % (self.IP_ADDRESS, relative_uri)
+
+
+def get_configured_signature_version(service_name, client_config,
```

Comment:
What was the reason for having this pulled out into ``utils`` as public functions as opposed to keeping a ``client`` module level internal function?